src/plugins/intel_cpu/src/graph.cpp

```diff
@@ -1414,7 +1414,7 @@ void Graph::DropNode(const NodePtr &node) {
             if (!child)
                 continue;
 
-            EdgePtr &remEdge = p_edge;
+            EdgePtr remEdge = p_edge;
```

Comment:
The code L1417-L1423 is not needed here from now and can be moved to the line L1409. Actually just `inNum` is required in this loop.
The same is applicable for fn `DropDWConvNode`

Reply:
Good point, thanks. Fixed